native/dgetf2.go

```diff
@@ -0,0 +1,64 @@
+package native
+
+import (
+	"math"
+
+	"github.com/gonum/blas/blas64"
+)
+
+// Dgetf2 computes the LU decomposition of the m×n matrix a.
+// The LU decomposition is a factorization of a into
+//  a = P * L * U
```

Comment:
I don't know if we do this elsewhere, but we probably should if not. "a" here should be "A" since we are not actually describing the abstract matrix in the parameter a as we do in mat64; m, n, a and lda are required to do this. So, leave the parameter as "a" and change the doc to "A" where appropriate. Probably we need to vet all the docs at some point to ensure we are doing this consistently.